rpcs3/Emu/system_config.h

```diff
@@ -96,6 +98,16 @@ struct cfg_root : cfg::node
 			return get(dev_flash);
 		}
 
+		std::string get_dev_flash2() const
+		{
+			return get(dev_flash2, "dev_flash2/");
+		}
+
+		std::string get_dev_flash3() const
+		{
+			return get(dev_flash3, "dev_flash3/");
```

Comment:
```suggestion
			return get(dev_flash3);
```

The need for this has been removed recently.

Reply:
Done